### PR TITLE
Feature: 78347 - Add StdWrap properties to FilesProcessor (10.3)

### LINKED ISSUE #338: Feature: 78347 - Add StdWrap properties to FilesProcessor (10.3)
| Name        | Value     |
| -------------- | ----------- |
| Type:         |  Feature |
| Merged in: | 10.3       |
| Gerrit:        | [62098](https://review.typo3.org/c/Packages/TYPO3.CMS/+/62098) |
| Changelog: | [Feature: 78347 - Add StdWrap properties to FilesProcessor](https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.3/Feature-78347-AddStdWrapPropertiesToFilesProcessor.html) |

### PR DESCRIPTION
**Pull request by linawolf**
merge into 10.4, cherry-pick into master

resolves #338

### CODE REVIEW

File: Documentation/ContentObjects/Fluidtemplate/DataProcessing.rst [hunk 1]
```diff
@@ -205,6 +205,31 @@ In the Fluid template then iterate over the files:
...
+      10 = TYPO3\CMS\Frontend\DataProcessing\FilesProcessor
+      10 {
+         references.data = levelmedia: -1, slide
+         as = myfiles
+      }
+   }
+
+The :php:`FilesProcessor` can slide up and down the rootline to collect
+images for FLUID templates. One usual feature is to use images
+attached to pages and use them up and down the page tree for
```
Comment (sypets) [inline Documentation/ContentObjects/Fluidtemplate/DataProcessing.rst:229]: Is "up and down the page tree" misleading? I would assume images are only looked for by going up the pagetree?

File: Documentation/ContentObjects/Fluidtemplate/DataProcessing.rst [hunk 2]
```diff
@@ -205,6 +205,31 @@ In the Fluid template then iterate over the files:
...
+header images in frontend.
```
Comment (sypets) [inline Documentation/ContentObjects/Fluidtemplate/DataProcessing.rst:230]: > in frontend

in the frontend

File: Documentation/ContentObjects/Fluidtemplate/DataProcessing.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/ContentObjects/Fluidtemplate/DataProcessing.rst]: You are looking for images up the pagetree, however the images get slided down the rootline. Graa!

**Review by sypets — APPROVED** [review]

File: Documentation/ContentObjects/Fluidtemplate/DataProcessing.rst (hunk 1 above)
Comment (sypets) [reply Documentation/ContentObjects/Fluidtemplate/DataProcessing.rst]: Yes, I understand. It is a bit difficult to explain.

**Pull request merged**